skimage/metrics/tests/test_structural_similarity.py

```diff
@@ -223,6 +223,18 @@ def test_mssim_mixed_dtype():
     assert_almost_equal(mssim, mssim_mixed)
 
 
+def test_structural_similarity_small_image():
+    X = np.zeros((5, 5), dtype=np.double)
```

Comment:
```suggestion
@pytest.mark.parametrize('dtype', [np.float16, np.float32, np.float64])
def test_structural_similarity_small_image(dtype):
    X = np.zeros((5, 5), dtype=dtype)
```
in light of #5220 /cc @grlee77 
@lilisako please merge `main` (into this branch) once #5220 is merged into `main`

Reply:
#5220 is now merged into `main` :)

Reply:
Thanks for letting me know! I just merged main into this branch and waiting for tests to pass!